### DISCUSSION TITLE
Where is the slice ID?

### DISCUSSION THREAD

**Discussion by ghost** [apache/superset#16578] [General]
Hi, I want to exclude  a couple of charts from filtering, but I can't figure out where can I find ID's of my charts.
I haven't found any information in the documentation, please assist!

**Comment by kamalkeshavani-aiinside** [comment]
Here is how your chart/slice explore url will look like:
http://localhost:8088/superset/explore/?form_data=%7B%22slice_id%22%3A%20XXX%7D

decoding the above url will look like
form_data={"slice_id": XXX}
